packages/docs/src/components/sidebar/sidebar.tsx

```diff
@@ -57,8 +83,16 @@ export function Items({
                 class={{
                   'is-active': pathname === item.href,
                 }}
+                style={{ display: 'flex' }}
               >
                 {item.text}
+                {markdownItems[item.href!]?.frontmatter?.updated_at ? (
+                  new Date(markdownItems[item.href!]?.frontmatter?.updated_at!).getTime() +
+                    5 * 24 * 60 * 60 * 1000 >
```

Comment:
Can you create a constant for this value?

Reply:
I have mode some changes, I think it's better now.